src/dbSta/src/dbSdcNetwork.cc

```diff
@@ -48,15 +48,13 @@ dbSdcNetwork::dbSdcNetwork(Network* network) : SdcNetwork(network)
 }
 
 // Override SdcNetwork to NetworkNameAdapter.
-Instance* dbSdcNetwork::findInstance(const char* path_name) const
-{
+Instance* dbSdcNetwork::findInstance(const char* path_name) const{
   Instance* inst = network_->findInstance(path_name);
   if (inst == nullptr)
     inst = network_->findInstance(escapeDividers(path_name, this));
   if (inst == nullptr)
     inst = network_->findInstance(escapeBrackets(path_name, this));
-  return inst;
-}
+  return inst;}
```

Comment:
I guess this is a test change but it should be removed from the PR.  Same with src/rsz/src/EstimateWireParasitics.cc

Reply:
I was waiting for the review to revert it